lib/gui/models/store.js

```diff
@@ -46,6 +46,7 @@ const DEFAULT_STATE = Immutable.fromJS({
     unmountOnSuccess: true,
     validateWriteOnSuccess: true,
     sleepUpdateCheck: false,
+    useUnstableUpdateChannel: false,
```

Comment:
Repeating this comment here, because it looks like you missed it last time:
If a user has never installed Etcher before, but then installs Etcher 1.0.0-rc.4 could we default useUnstableUpdateChannel to being true? And conversely if a user has never installed Etcher before, but then installs Etcher 1.0.0 could we default useUnstableUpdateChannel to being false?